CMakeLists.txt

```diff
@@ -137,8 +137,8 @@ set_property(CACHE CMAKE_BUILD_TYPE PROPERTY
 # version string, else they are combined with the result of `git describe`.
 set(NVIM_VERSION_MAJOR 0)
 set(NVIM_VERSION_MINOR 5)
-set(NVIM_VERSION_PATCH 0)
-set(NVIM_VERSION_PRERELEASE "") # for package maintainers
+set(NVIM_VERSION_PATCH 2)
```

Comment:
Minor question, why is this 2?

Reply:
I think this is for _after_ the release? (So a build from `release-0.5` reports itself as `0.5.2-dev` -- see the following line.)

Reply:
it should have been 1 before.

Reply:
Got it, thanks!

Reply:
because I dun goofed